kotlin-emotion/src/main/kotlin/emotion/css/keyframes.kt

```diff
@@ -0,0 +1,14 @@
+@file:JsModule("@emotion/css")
+@file:JsNonModule
+
+package emotion.css
+
+import csstype.AnimationName
+import csstype.Percentage
+import csstype.Properties
+import kotlinx.js.Record
+
+@JsName("keyframes")
+external fun rawKeyframes(
+    blocks: Record<Percentage, Properties>,
```

Comment:
`block`?
`Record<String, Properties>`?

Reply:
I used `blocks` because in MDN specs this fragment is named `<keyframe-block-list>`, do you think just `block` would be better?
`Record<String, Properties>` is incorrect alias, according to specs only these values are available `<keyframe-selector> = from | to | <percentage>`, I have supported `from` and `to` additionally

Reply:
`blocks` is renamed